cypress/fixtures/experimental/dateInput.json

```diff
@@ -0,0 +1,72 @@
+{
+  "disabled": {
+    "disabled": true
+  },
+  "enabled": {
+    "disabled": false
+  },
+  "readOnly": {
+    "readOnly": true
+  },
+  "readOnlyFalse": {
+    "readOnly": false
+  },
+  "fieldHelpOtherLanguage": {
+    "fieldHelp": "mp150ú¿¡üßä"
+  },
+  "fieldHelpSpecialCharacter": {
+    "fieldHelp": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "labelOtherLanguage": {
+    "label": "mp150ú¿¡üßä"
+  },
+  "labelSpecialCharacter": {
+    "label": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "labelInline": {
+    "label": "label",
+    "labelInline": true
+  },
+  "directionLeft": {
+    "labelAlign": "left",
+    "label": "label",
+    "labelHelp": "labelHelp",
+    "labelInline": true
+  },
+  "directionRight": {
+    "labelAlign": "right",
+    "label": "label",
+    "labelHelp": "labelHelp",
+    "labelInline": true
+  },
+  "labelWidth0": {
+    "label": "label",
+    "labelInline": true,
+    "labelWidth": 0
+  },
+  "labelWidth10": {
+    "label": "label",
+    "labelInline": true,
+    "labelWidth": 10
+  },
+  "labelWidth100": {
+    "label": "label",
+    "labelInline": true,
+    "labelWidth": 100
+  },
+  "sliderWidth0": {
+    "label": "label",
+    "labelInline": true,
+    "labelWidth": 0
+  },
+  "sliderWidth10": {
```

Comment:
what is the difference between this object and `labelWidth10` above ? Seems to me the same parameters

Reply:
removed